src/utils.rs

```diff
@@ -57,3 +57,18 @@ pub fn effective_value(output: &OutputGroup, feerate: f32) -> u64 {
         .value
         .saturating_sub(calculate_fee(output.weight, feerate))
 }
+
+// Returns the weight of the output group(s)
+pub fn calculate_base_weight_btc(output_weight: u64) -> u64 {
```

Comment:
This `output_weight` variable denotes the weight of `Vec<TxOut>` in the `Transaction`.